boot/zephyr/flash_map_extended.c

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ * Copyright (c) 2015 Runtime Inc
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <flash.h>
+
+#include "target.h"
+
+#include <flash_map_backend/flash_map_backend.h>
+#include <hal/hal_flash.h>
+#include <sysflash/sysflash.h>
+
+#define BOOT_LOG_LEVEL BOOT_LOG_LEVEL_INFO
```

Comment:
Please remove this line, eventually #273 will be merged and I prefer not to rework this file after that...